install.sh

```diff
@@ -96,7 +96,13 @@ sudo apt-get -qq update
 fancy_message info "Installing packages"
 sudo apt-get install -qq -y {curl,wget,stow,build-essential,unzip,tree,dialog}
```

Comment:
```
sudo apt-get install -qq -y {curl,wget,stow,build-essential,unzip,tree,dialog} 2>/dev/null
```

This would suppress warning of apt regarding stable CLI interface

Reply:
It's using `apt-get` not `apt`, so there will be no warnings printed.

Reply:
See output of remote bash in tests.

Reply:
I misunderstood the base install part as pacstall installer